Platform/MeteorlakeBoardPkg/Library/FspsUpdUpdateLib/FspsUpdUpdateLib.c

```diff
@@ -842,6 +842,14 @@ UpdateFspConfig (
   FspsConfig->SerialIoSpiMosiPinMux[0] = 0x14a48d;
   FspsConfig->DmiPowerGatingDis = 0x0;
 
+  switch (GetPlatformId ()) {
+    case BoardIdMtlPUpXtremei14:
+      FspsConfig->GnaEnable = 0x0;
+      break;
+    default:
+      break;
```

Comment:
Often if we need set a fixed value for a given platform, we would add a SBL config data for this UPD and use the SBL config data to update FSP UPD. In this example, add sbl config data GnaEnable with default value 1 so that it would not impact other platforms. and its value for this Upx could be set to 0.